aten/src/ATen/core/ivalue.cpp

```diff
@@ -567,6 +573,8 @@ std::ostream& IValue::repr(
     }
     case IValue::Tag::Int:
       return out << v.toInt();
+    case IValue::Tag::BoxedInt:
+      return out << "BoxedInt(" << v.toInt() << ")";
```

Comment:
this doesn't look quite right, this should use the printer for BoxedInt, not int (which will be some garbage)